bstats-bukkit-lite/src/main/java/org/bstats/bukkit/MetricsLite.java

```diff
@@ -111,7 +107,8 @@ public MetricsLite(Plugin plugin) {
             ).copyDefaults(true);
             try {
                 config.save(configFile);
-            } catch (IOException ignored) { }
+            } catch (IOException ignored) {
```

Comment:
While there's not anything particularly wrong with this layout, I don't think it's something that should be changed.

Reply:
yeah, i missed that, i did an automatic codestyle cleanup based on the general codestyle